content/md/en/docs/tutorials/connect-relay-and-parachains/open-message-passing-channels.md

```diff
@@ -76,7 +80,7 @@ Before the parachain can exchange messages with another parachain, it must have
 
 5. Add the address and a name for parachain A (1000), then click **Save**.
 
-6. Click **Accounts** and transfer some tokens from Alice to the parachain A (1000) account.
+6. Click **Accounts** and transfer some units from Alice to the parachain A (1000) account.
```

Comment:
Some units of what? Best to clarify here.

Reply:
assets instead of tokens or units